src/components/Pages/Login/LoginForm/index.js

```diff
@@ -80,22 +107,42 @@ export const LoginForm = () => {
       <form onSubmit={handleSubmit}>
         <div className={style.form__container}>
           <h1>LOGIN</h1>
+
           <input
             type="email"
-            // required
+            required
             // pattern="/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/"
+            value={email}
+            className={
```

Comment:
we will check a package that allows you to add more classes, instead of repeating css code inside each class - classnames